src/main/java/org/broadinstitute/hellbender/utils/param/ParamUtils.java

```diff
@@ -105,4 +105,27 @@ public static double isFinite(final double val, final String message) {
         }
         return val;
     }
+
+    /**
+     * log the value in base b
+     *
+     * @param val -- The value to be logged.
+     * @param base -- The base of the logging
+     * @return the logged value.  If val is zero, returns Double.NEGATIVE_INFINITY.  If base is zero, returns -0.0.  If any input
+     *  is negative, returns Double.NaN.
+     */
+    public static double logb(final double val, final double base) {
+        return Math.log(val)/Math.log(base);
+    }
+
+    /**
+     * log the value in base 2
+     *
+     * @param val -- The value to be logged.
+     * @return the logged value.  If val is zero, returns Double.NEGATIVE_INFINITY.  If val
+     *  is negative, returns Double.NaN.
+     */
+    public static double log2(final double val) {
+        return Math.log(val)/Math.log(2.0);
```

Comment:
I leave it up to you to decide whether this will be used heavily enough to justify caching 1/log(2).